handbook/engineering/search/iteration_log.md

```diff
@@ -14,9 +14,41 @@ This document contains the goals and work log for the search team's [2-week iter
 
 ## 2020-09-21 to 2020-10-02
 
-### $GOAL_OR_THEME
+### Scale indexed search to 500k repositories
 
-- **Owner(s):** $OWNER
+- **Owner(s):** Stefan
+- **Outcomes:**
+    - $DESIRED_OUTCOME
```

Comment:
@stefanhengl can you fill in what you plan to accomplish with your work on performance in the next iteration?